styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-color: pink;
+  text-align: left;
+  font-family: "Lato", sans-serif;
+}
+h2 {
+  font-family: "Poppins", sans-serif;
+  line-height: 25px;
+}
+h3 {
+  font-size: 24px;
+  line-height: 18px;
+}
+
+h2,
+h3,
+p {
+  color: #5c585a;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 12px;
+}
+
+button {
+  background-color: white;
+  border: none;
+  bottom: 30px;
+  color: white;
+  cursor: pointer;
+  left: 140px;
+  margin: 5px;
+  padding: 12px 14px;
+  position: relative;
+}
+
+.container {
+  background: white;
+  border-radius: 10px;
+  height: 520px;
```

Comment:
In general, if you can avoid setting a `height` attribute on containers with text in them, that is best. It's very easy for more text to be added to something and then for your layout to break. 🙈